src/javascript/app/base/binary_pages.js

```diff
@@ -1,9 +1,10 @@
 // ==================== _common ====================
-const TabSelector = require('../../_common/tab_selector'); // eslint-disable-line import/order
+// eslint-disable-line import/order
 
 // ==================== app ====================
 const LoggedInHandler         = require('./logged_in');
 const Redirect                = require('./redirect');
+const TabSelector             = require('../../_common/tab_selector');
```

Comment:
why move this inside the app section? it has it's own _common section above

Reply:
It's a typo. I'll change this back.